pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1311,7 +1311,7 @@ func testUDPReusePort(t *testing.T, udpnet string, ip string) {
 
 	tr := setupTracer(t, cfg)
 
-	port := rand.Intn(32768) + 32768
+	port := 0
```

Comment:
if you're only using this once below you can just pass `0` to `createReuseServer()`